monai/apps/nnunet/nnunetv2_runner.py

```diff
@@ -136,7 +230,7 @@ def convert_dataset(self):
                 os.makedirs(os.path.join(raw_data_foldername, "imagesTr"))
                 os.makedirs(os.path.join(raw_data_foldername, "labelsTr"))
             else:
-                logger.warning("Input '.json' data list is incorrect.")
+                logger.warning("The datalist file has incorrect format: the `training` key is not found.")
```

Comment:
There might be other errors.

Reply:
If there are other errors, should we put more checks besides the training key?

Reply:
Since there is no reply, I consider this is not an urgent issue to deal with and close the conversation. @dongyang0122 please feel free to comment or open another issue.